src/Pinch/Transport.hs

```diff
@@ -88,7 +83,7 @@ unframedTransport c = do
       (leftOvers, r) <- runGetWith getSome p bs'
```

Comment:
Not necessarily part of this PR, but I guess the `getSome` calls on line 87/82 is not strictly necessary? Noticed it know due to the changes to the framedTransport.

Reply:
I applied a similar change here too.